tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetFramework.cs

```diff
@@ -31,7 +31,17 @@ private static string ResolveManagedProfilerDirectory()
 
         private static Assembly AssemblyResolve_ManagedProfilerDependencies(object sender, ResolveEventArgs args)
         {
-            return ResolveAssembly(args.Name);
+            try
+            {
+                StartupLogger.Debug("Assembly Resolve event received for: '{0}'", args.Name);
```

Comment:
I don't remember why, but in ResolveAssembly there is this comment:

```
WARNING: Logs must not be added _before_ we check for the above bail-out conditions
```

Just to be safe, I believe you should remove this log

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/5129/commits/1abe99b180ab9c2bb3986c8c4af85c78ab0ae4bf